_includes/tutorials/schedule-ktable-ttl-aggregate/kstreams/markup/dev/intro.adoc

```diff
@@ -0,0 +1,9 @@
+In a previous tutorial, we showed how to purge KTables which are created directly from an input topic. See https://kafka-tutorials.confluent.io/schedule-ktable-ttl/kstreams.html[Expire data in a KTable with TTLs]
```

Comment:
1. This is a nice addition to cross link to the existing tutorial.  Can you do the equivalent cross-linking from that tutorial too?  http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/kafka-tutorials-ii/schedule-ktable-ttl/kstreams.html
2. In all other tutorials, this kind of description is usually at the top under `Use case`, and the first step in the tutorial is not description but actual step/action for a user to take.  So I think this entire bulk of text should be moved to the top, for consistency.
3. Can you please elaborate on the end use case (e.g. a business context) when a user would want to do this?

Reply:
Have moved it to the top now